docs/DeveloperGuide.md

```diff
@@ -695,3 +695,77 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Mass Operations [Proposed]
+
+#### What is the feature about
+Provides a way for users to mark and unmark multiple `Task` at a time. The current implementation of mark and unmark
+users have to mark and unmark multiple tasks as completed or uncompleted individually, one at a time.
+
+#### How the feature is implemented
+
+The feature is to be implemented with the addition of a `MassOpsParser` class which parses through user inputs
+consisting of multiple indexes and processes the indexes to return an `ArrayList` of `Indexes` for `MarkCommand` and
+`UnmarkCommand` to execute on.
+
+##### MassOps: Marking tasks
```

Comment:
Should we use short-forms or should we spell out the whole command? i.e. `MassOperations`